alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/metadataExtractors/DWGMetadataExtractor.java

```diff
@@ -64,13 +65,12 @@ public DWGMetadataExtractor()
         super(logger);
     }
 
-    @SuppressWarnings("deprecation")
     @Override
     protected Map<String, Serializable> extractSpecific(Metadata metadata,
                                                         Map<String, Serializable> properties, Map<String,String> headers)
     {
-        putRawValue(KEY_KEYWORD, metadata.get(Metadata.KEYWORDS), properties);
-        putRawValue(KEY_LAST_AUTHOR, metadata.get(Metadata.LAST_AUTHOR), properties);
+        putRawValue(KEY_KEYWORD, metadata.get(TikaCoreProperties.SUBJECT), properties);
```

Comment:
SUBJECT to KEYWORD - is that correct?

Reply:
According to the [migration guide ](https://cwiki.apache.org/confluence/display/TIKA/Migrating+to+Tika+2.0.0) the `KEYWORD` has been transitioned to `dc:subject`